Wireframe/index.html

```diff
@@ -10,20 +10,70 @@
     <header>
       <h1>Wireframe</h1>
       <p>
-        This is the default, provided code and no changes have been made yet.
+<<<<<<< HEAD
+        This is a fork from CodeYourFuture which I add three articles
+=======
+        Fork of CodeYourFuture Wireframe with a placeholder and articles.
+>>>>>>> main
       </p>
     </header>
+
     <main>
-      <article>
-        <img src="placeholder.svg" alt="" />
-        <h2>Title</h2>
-        <p>
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam,
-          voluptates. Quisquam, voluptates.
-        </p>
-        <a href="">Read more</a>
-      </article>
+      <div class="top">
+        <section class="article">
```

Comment:
Can you review the exercise requirements? I don't see 'placeholder' as one of the articles that should be displayed in your webpage.